src/main/java/uk/gov/ons/ctp/response/collection/exercise/distribution/SampleUnitDistributor.java

```diff
@@ -105,16 +105,13 @@ public void distributeSampleUnits(CollectionExercise exercise) {
       }
 
     } catch (LockingException | CTPException ex) {
-      log.error("Distribution failed due to {}", ex.getMessage());
-      log.error("Stack trace: " + ex);
+      log.with("message", ex.getMessage()).error("Distribution failed", ex);
```

Comment:
I should have commented on this before... but arguably logging out the exception message is unnecessary, because the logging framework should output it

Reply:
I agree kind of just left it in because i wasn't quite sure why there were two logs one with the full exception and then one specifically the message. Will remove the exception message.